src/main/java/org/springframework/data/jpa/repository/support/CrudMethodMetadataPostProcessor.java

```diff
@@ -85,7 +86,7 @@ public CrudMethodMetadata getLockMetadataProvider() {
 	 * @author Oliver Gierke
 	 * @author Thomas Darimont
 	 */
-	static enum CrudMethodMetadataPopulatingMethodIntercceptor implements MethodInterceptor {
+	static enum CrudMethodMetadataPopulatingMethodInterceptor implements MethodInterceptor {
```

Comment:
The `RepositoryProxyPostProcessor` creating this `MethodInterceptor` already knows about a `RepositoryInformation`. If we turned the enum into a class, we could actually wire the `EntityInformation` contained in the `RepositoryInformation` to the instance about to be created and thus determine the default entity graph name internally without having the `SimpleJpaRepository` manually assemble it by obtaining information from the `CrudMethodMetadata` and then eventually forwarding to the `JpaEntityGraph`.